src/template.njk

```diff
@@ -19,9 +19,9 @@
     {% endblock %}
 
     {% block head %}{% endblock %}
-
     {# The default og:image is added below head so that scrapers see any custom metatags first, and this is just a fallback #}
-    <meta property="og:image" content="{{ assetPath | default('/assets') }}/images/govuk-opengraph-image.png">
+    {# image url needs to be absolute e.g. http://wwww.domain.com/.../govuk-opengraph-image.png #}
+    <meta property="og:image" content="{{ assetUrl | default('/assets') }}/images/govuk-opengraph-image.png">
```

Comment:
I think the default needs to be 'assetPath' otherwise this is a breaking change still?

Reply:
maybe `{{ openGraphImagePath | assertPath | default('/assets') }}`

not sure

Reply:
it doesn't work with assetPath anyway, so not sure it's a breaking change

Reply:
Fair point, but consider if someone set their 'assetPath' to a absolute path, it would be working for those users?

Reply:
modified so that then default value for assetUrl is assetPath, insuring there it is not a breaking change if anyone specified an absolute url for the assets